libpermian/plugins/anaconda_webui/__init__.py

```diff
@@ -236,11 +241,23 @@ def setup(self):
         if self.canceled:
             return
         self.reportResult(Result('started', None, False))
+
         if not self.dryRun:
             self._start_vm()
+            self.log('Waiting for IP', show=True)
+            vm_ip = self.hypervisor.wait_for_ip(self.vm_name)
+
+            if self.hypervisor.remote:
+                # Configure port forwarding
+                self.port_ssh, self.port_webui = self.hypervisor.configure_forwarding(vm_ip, [self.port_ssh, self.port_webui])
+                # After port forwarding the VM is access via localhost / host of the container
+                self.test_system_ip = 'host.containers.internal' if self.use_container else 'localhost'
+                self.webui_url = f'http://localhost:{self.port_webui}{self.webui_location}'
```

Comment:
Shouldn't it be `host.containers.internal` instead of `localhost` when executed in container?

Reply:
self.webui_url is used only be the workflow to check if the webui is running, it is not used from inside the container.

Reply:
Oh, right, that makes sense although it's kind of confusing as one is used only from outside of the container while the other one may or may not be used inside it.